eng/intellisense.targets

```diff
@@ -0,0 +1,41 @@
+<Project>
+
+  <PropertyGroup>
+    <XmlDocFileRoot>$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', 'microsoft.private.intellisense', '$(MicrosoftPrivateIntellisenseVersion)', 'IntellisenseFiles'))</XmlDocFileRoot>
+  </PropertyGroup>
+
+  <!-- If the project is not generating its own documentation yet, retrieve it from the dotnet-api-docs nuget package. -->
+  <Choose>
+    <When Condition="('$(UseIntellisenseDocumentationFile)' == '' or '$(UseIntellisenseDocumentationFile)' == 'false') and Exists($([MSBuild]::NormalizePath('$(XmlDocFileRoot)', 'net', '1033', '$(AssemblyName).xml')))">
```

Comment:
Please avoid these duplicated NormalizePath msbuild functions as they aren't cost free. You could instead define another two properties next to `XmlDocFileRoot´ for that purpose:

```xml
<XmlDocFileFromNetFolder>$([MSBuild]::NormalizePath('$(XmlDocFileRoot)', 'net', '1033', '$(AssemblyName).xml')</XmlDocFileFromNetFolder>
<XmlDocFileFromDotNetPlatExtFolder>$([MSBuild]::NormalizePath('$(XmlDocFileRoot)', 'dotnet-plat-ext', '1033', '$(AssemblyName).xml')</XmlDocFileFromDotNetPlatExtFolder>
```